scripts/prepare

```diff
@@ -19,15 +19,16 @@ for f in packages/*; do
 			rm -rf ${f}/charts
 			mkdir -p ${f}/charts
 			if [[ $type == 'git' ]]; then
+				rm -rf /tmp/tmp-charts
 				mkdir -p /tmp/tmp-charts
 				git clone --depth=1 --no-tags $url /tmp/tmp-charts
 				pwd=$(pwd)
 				cd /tmp/tmp-charts
+				git fetch --tags
```

Comment:
I thought pulling tags was removed to improve performance.

Reply:
Hmm I added this is because it was related to the CI build issues I found [here](https://github.com/rancher/charts/pull/641#issuecomment-691610987); CI fails because the tag was not found locally.

Since this is probably just an issue with adding support for tags, I've removed this line from this PR and modified my `package.yaml` to use a literal commit rather than a tag for this. Thinking more about this, I also think this is the right decision since the base that a tag is referring to could be modified on a later push, so the patch might fail as a result.

cc: @StrongMonkey for this issue with specifying tags for Git-based `package.yaml`s